### ISSUE TITLE
Произошла непредвиденная ошибка Нажмите на кнопку "Ошибка", чтобы отправить нам детали

### ISSUE THREAD

**Issue by KOT040188** [clipto-pro/Desktop#134]
Тут вроде по-русски можно? Что это за ошибка как её убрать? Opensuse tumbleweed.
![ss_2022 04 27_20:49:43](https://user-images.githubusercontent.com/11147600/165588831-06bcae8b-f0c6-4739-ab8c-ad520809b457.png)

**Comment by atrashler** [comment]
Кликните на нее и она должна исчезнуть. Помогло?

**Comment by KOT040188** [comment]
> Кликните на нее и она должна исчезнуть. Помогло?

Нет, при нажатии открывается бесполезная информация. Приперезапуске снова появляется.

**Comment by atrashler** [comment]
Тогда лучше переустановить. Последняя версия эта -
https://github.com/clipto-pro/Desktop/releases/tag/v7.2.17

Важно именно, чтобы кэш приложения почистился с компа. Похоже на глюк
системы - приложение не может получить коннект, но не знает почему.

**Comment by KOT040188** [comment]
> Тогда лучше переустановить. Последняя версия эта

Да я в курсе, ее и скачал в формате appimage, тут переустанавливать нечего)

>Важно именно, чтобы кэш приложения почистился с компа

Где он лежит?

**Comment by atrashler** [comment]
Немного задержался:)

Вот - https://clipto.pro/l/f91nrrark2saJahaA


   1. Clean local cache of the app
      - on *Windows* delete the folder
      *C:\Users<USER>\AppData\Roaming\Clipto*
      - on *macOS* delete the folder */Users//Library/Application
      Support/Clipto*
      - on *Linux* delete the folder */home//.config/Clipto*

**Comment by KOT040188** [comment]
Помогло, спасибо.
